tests/test_rdf.py

```diff
@@ -41,16 +40,6 @@ def test_namespacing():
     assert str(r.uri('bar', 'abc')) == 'abc#bar'
 
 
-def test_namespacing():
```

Comment:
`test_get_applications` uses the same rdf file but it doesn't validate the uri or namespace or am I missing something?